Add php 7 and 8

b1rdex opened the pull request:
Had to drop php 5.3

b1rdex commented:
Here is build log: https://travis-ci.com/b1rdex/phpiredis/builds/105300525

b1rdex commented:
Here is last build log with CFLAGS set: https://travis-ci.com/b1rdex/phpiredis/builds/105302618

b1rdex commented:
The build is green: https://travis-ci.com/github/b1rdex/phpiredis/builds/177896588

nrk commented:
Hi @b1rdex, just wondering: would it be possible to keep tests running on PHP 5.3, 5.4 and 5.5 on Travis CI [just like we did with Predis](https://github.com/predis/predis/blob/v1.1.3/.travis.yml#L19-L26)? We could also [add PHP 8.0](https://github.com/predis/predis/blob/v1.1.3/.travis.yml#L33) while we are at this, planning to merge #66 in a few days.

b1rdex commented:
@nrk I've added 5.3, 5.4 and 5.5 back and added 8.0. The build passed (https://travis-ci.com/github/b1rdex/phpiredis/builds/189732401) except for 8.0. Then I decided to update the branch and rebase it over v1.1. And now it's broken: https://travis-ci.com/github/b1rdex/phpiredis/builds/189734122. I don't get why and can't fix it. What should I do next? 😄 

Also, could you please fix Travis integration? Would be easier to see the build status here on GitHub.

b1rdex committed:
Add php 7 and php 8

b1rdex commented:
Actually, v1.1 branch errors too: https://travis-ci.com/github/b1rdex/phpiredis/builds/189735956
